app/main/machmakin.tsx

```
@@ -1,25 +1,288 @@
+};
+
+// Composant pour les points animés
+const AnimatedDots = () => {
+  const dots = [0, 1, 2];
+
+  return (
+    <View style={styles.dotsContainer}>
+      {dots.map((index) => {
+        const scale = useSharedValue(1);
+
+        React.useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)